lua/grapple-utils.lua

```diff
@@ -22,6 +13,58 @@ local function get_current_index(tags, current_path)
 	return nil
 end
 
+function M.move_to_index(opts)
```

Comment:
This function is very similar to just using `require("grapple").tag({index = index})`.
The difference is that this function supports indices that `grapple` considers out of bounds using modulo.
I think the name should reflect this modulo behaviour.